scripts/startup/README.md

```diff
@@ -15,8 +19,55 @@ sudo chmod +x /etc/profile.d/set_panoptes_env_vars.sh
 This will then be executed when any user logs in. The exact means
 of doing this varies on different versions of Unix/Linux.
 
-Next, we need to arrange for `su_panoptes.sh` to be executed when
-the operating system launches. First copy it to /etc:
+If you've changed the location of the PANOPTES directories (e.g. if
+log files are not stored in $PANDIR/logs), you'll need to edit that
+script accordingly.
+
+# Start PANOPTES Software on System Start
+
+We intend for a PANOPTES telescope to run automatically, including after
+reboots. We document these ways to achieve this:
+
+* `@reboot` rule in crontab of user $PANUSER
+* Execution by root during boot
+
+## Crontab
+
+[Crontab](https://linux.die.net/man/5/crontab) provides a means to run
+a command line at various times. Some versions of crontab have support
+for [`@reboot`](https://www.google.com/search?q=crontab+%40reboot) as
+a *time* at which to run a command. If your system has this feature, it
+is the easiest way to launch the PANOPTES software.
+Add these lines to your ($PANUSER) crontab by executing the command
+`crontab -e`, which will start an editor on your crontab file.
+
+```bash
+POCS=/var/panoptes/POCS
+PANLOG=/var/panoptes/logs
+
+# m h  dom mon dow   command
+@reboot              /bin/bash --login $POCS/scripts/startup/tmux_launch.sh
+*/5 *   *   *   *    /bin/bash --login python $POCS/scripts/plot_weather.py >> $PANLOG/plot_weather.log 2>&1
+11  12  *   *   *    /bin/bash --login python $POCS/pocs/utils/data.py >> $PANLOG/update_data.log 2>&1
```

Comment:
The other wiki page about this only has it running once per week (i.e. `dow 0`). Did you mean for it to run every day? Doesn't hurt I suppose but not totally necessary.